batch7crud-roll2/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,305 @@
+body, html {
+    height: 100%;
+    position: relative;
+}
+
+.mainWrapper {
+    max-width: 100%;
+}
+
+.headerWrapper {
+    background-color: #666666;
+    padding: 30px 45px 30px 100px;
+    color: #ffffff;
+    border-bottom: 2px solid #333333;
+}
+
+.logo {
+    float: left;
+    font-size: 22px;
+}
+
+.bodyWrapper {
+    min-height: 600px;
+}
+
+.col2-left {
+    width: 100%;
+}
+
+.left {
+    width: 20%;
+    background-color: #336699;
+    min-height: 850px;
+    float: left;
+}
+
+.right {
+    float: left;
+    width: 80%;
+    min-height: 700px;
+}
+
+/*Login Page*/
+.loginBox {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename selector loginBox to match the regular expression: ^[a-z][a-z\-]*$ [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Aselector-naming-convention)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)